CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased
+* Add done boolean flag to data passed to the document template service ([#1049](https://github.com/opendevstack/ods-jenkins-shared-library/pull/1049))
```

Comment:
Better use the number of associated issue #1048

Reply:
Done in last commit